integration/110_shutdown_test.sh

```diff
@@ -5,12 +5,15 @@
 start_suite "Check scope exits cleanly within 10 seconds"
 
 scope_on $HOST1 launch
+sleep 10
```

Comment:
Does this piece of string have to be as long as 10? I'd try it with 3.

(likewise for the one below)